services/net/extract-quotes/LLM/Models/LLMResponse.cs

```diff
@@ -0,0 +1,63 @@
+using System.Text.Json.Serialization;
+
+namespace TNO.Services.ExtractQuotes.LLM.Models;
+
+/// <summary>
+/// Response from the LLM API
+/// </summary>
+public class LLMResponse
```

Comment:
It's not critical for this PR, but the rule of thumb is a file for each class.  Having separate files makes it easier to maintain over time.